release-team/hack/sync-enhancements-github-project-beta.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: `sync-enhancements-github-project-beta.sh`
+
+set -e
+set -u
+set -o pipefail
+
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_ROOT="$(cd "${SCRIPT_DIR}/.." && pwd )"
+TMPDIR="${TMPDIR:-${REPO_ROOT}/tmp}"
+
+MILESTONE_ISSUES_JSON="${TMPDIR}/milestone_issues.json"
+mkdir -p "$(dirname "${MILESTONE_ISSUES_JSON}")"
+
+
+ORGANIZATION='kubernetes'
+REPOSITORY='enhancements'
+PROJECT_NUMBER=${GITHUB_PROJECT_BETA_NUMBER}
+MILESTONE=${RELEASE_MILESTONE}
+
+milestone_issue_ids=()
+milestone_issue_numbers=()
+
+
+# function definitions
+function get_field_ids_from_github_beta_project() {
+    if [ $1 == 'project-id' ]
+    then
+        query='.data.organization.projectV2.id'
+    else
+        ##for 'issue-number-field-id'
+        query='.data.organization.projectV2.fields.nodes[] | select(.name== "Issue Number") | .id'
+    fi
+    
+    ID="$( gh api graphql -f query='
+                    query($org: String!, $number: Int!) {
+                    organization(login: $org){
+                        projectV2(number: $number) {
+                        id
+                        fields(first:100) {
+                            nodes {
+                            ... on ProjectV2Field {
+                                id
+                                name
+                            }
+                            ... on ProjectV2SingleSelectField {
+                                id
+                                name
+                                options {
+                                    id
+                                    name
+                                }
+                            }
+                            }
+                        }
+                        }
+                    }
+                    }' -f org=${ORGANIZATION} -F number=${PROJECT_NUMBER} --jq "$query")"
+    echo $ID
```

Comment:
`echo $ID` is working as a return statement in this case (equivalent to `return $ID` in other programming languages).
So, it was intended that no text be added around the `$ID`.

Because the `return` command in bash only returns the function's exit code and not the actual value, this is a workaround.